### PR TITLE
updated endpoints to match server-side.

### PR DESCRIPTION
**Pull request by ChineduOzodi**
mapping the endpoints to server side. This breaks the ability to use the mock data client side since it takes more work to get custom endpoints created the mock data. I did replaced the mapping to server-side "/" to "-" to allow for compatibility and easy switching if the client side mock data is ever needed again.

### CODE REVIEW

File: src/app/routes/curriculum-editor/create-curriculum/create-curriculum.component.ts [hunk 1]
```diff
@@ -150,6 +155,10 @@ export class CreateCurriculumComponent implements OnInit {
           // Save the next week
           const weekNum = week.weekNum + 1;
           this.saveWeek(curriculum, weekNum);
+        } else {
+          // done with all days and weeks, refresh page
+          console.log('Resfreshing page');
```
Comment (MarcinSalamon) [inline src/app/routes/curriculum-editor/create-curriculum/create-curriculum.component.ts:171]: delete, keep this commit short

File: src/app/routes/curriculum-editor/curriculum-editor.component.ts [hunk 2]
```diff
@@ -207,6 +208,14 @@ export class CurriculumEditorComponent implements OnInit {
         }
       }
     );
+    dialogRef.afterClosed().subscribe(curriculum => {
+      console.log('new curriculum');
```
Comment (MarcinSalamon) [inline src/app/routes/curriculum-editor/curriculum-editor.component.ts:247]: delete

File: src/environments/environment.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ export const environment = {
   production: false,
   envName: `dev`,
   context: `http://localhost:4200`,
-  apiUrl: 'http://localhost:3000/',
+  apiUrl: 'http://localhost:8765/bam/',
```
Comment (MarcinSalamon) [inline src/environments/environment.ts:9]: we already have zuul url

**Review by MarcinSalamon — CHANGES_REQUESTED** [review]

**Review by ChineduOzodi — COMMENTED** [review]
Looks good

**Pull request merged**